SmartDeviceLink/SDLMenuManager.m

```diff
@@ -329,16 +484,29 @@ - (void)sdl_updateIdsOnMenuCells:(NSArray<SDLMenuCell *> *)menuCells parentId:(U
 }
 
 #pragma mark Commands / SubMenu RPCs
-
-- (NSArray<SDLRPCRequest *> *)sdl_mainMenuCommandsForCells:(NSArray<SDLMenuCell *> *)cells withArtwork:(BOOL)shouldHaveArtwork {
+/**
+ This method will receive the cells to be added and the updated menu array. It will then build an array of add commands using the correct index to position the new items in the correct location.
+
+ @param cells that will be added to the menu, this array must contain only cells that are not already in the menu.
+ @param shouldHaveArtwork artwork bool
+ @param the new menu array, this array should contain all the values the develeoper has set to be included in the new menu. This is used for placing the newly added cells in the correct locaiton.
```

Comment:
This param documentation is missing the name of the parameter `menu`, so I'm getting a documentation warning.

Reply:
fixed!